web/src/components/HomepageLaunchpad.tsx

```
@@ -0,0 +1,133 @@
+import { Link, LinkProps } from "@tanstack/react-router";
+import clsx from "clsx";
+import { Cog, ListCheck, Map, QrCode, Trophy, UsersRound } from "lucide-react";
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)